gateway/gateway.go

```diff
@@ -357,6 +358,8 @@ func (gateway *HandleT) userWebRequestWorkerProcess(userWebRequestWorker *userWe
 				misc.IncrementMapByKey(sourceFailStats, sourceTag, 1)
 				continue
 			}
+			gateway.logger.Debugf("event size from http request : %d", len(body))
```

Comment:
Remove this log. This will make it very difficult to view the logs.

Reply:
done